### DISCUSSION TITLE
Drop Support for ES2018

### DISCUSSION THREAD

**Discussion by sxzz** [element-plus/element-plus#9480] [RFC]
- Target Version: 3.x

# Summary

Drop support for ES2018. The minimum supported browser is ES2019.

# Motivation

We should give up some baggage every year, to better support modern browsers.

Target browsers should be updated to `year - 3`. It's 2022 now, so the target should be updated to `es2019`

ES2019: Chrome 73, Firefox 64, Edge 79 (not changed), Safari 12.1

# Reference

- https://262.ecma-international.org/10.0/
- [caniuse](https://caniuse.com/?feats=array-flat,mdn-javascript_builtins_object_fromentries,mdn-javascript_builtins_string_trimstart,mdn-javascript_builtins_symbol_description,mdn-javascript_statements_try_catch_optional_catch_binding,mdn-javascript_builtins_json_stringify_well_formed_stringify,mdn-javascript_builtins_array_sort_stable,mdn-javascript_builtins_function_tostring_tostring_revision)

**Comment by kooriookami** [comment]
We are now support latest 2 versions. So close this.
